tbot/machine/channel/channel.py

```diff
@@ -28,12 +28,15 @@
 import typing
 
 import tbot
+import tbot.error
 
 ChanIO = typing.TypeVar("ChanIO", bound="ChannelIO")
 
 
-class ChannelClosedException(Exception):
-    pass
+# compatibility aliases
+ChannelClosedException = tbot.error.ChannelClosedError
+ChannelBorrowedException = tbot.error.ChannelBorrowedError
+ChannelTakenException = tbot.error.ChannelTakenError
```

Comment:
I'll at some point mark those aliases deprecated (= emits a warning), but let's not go that far right now.